wvd-templates/DSC/Script-TestRegisterSessionHost.ps1

```diff
@@ -87,8 +87,9 @@ if (!$SessionHost) {
     Write-Log -Err "SessionHost '$SessionHostName' does not exist in Host Pool '$HostPoolName' in Tenant '$TenantName'"
     return $false
 }
-if ($SessionHost.Status -ne 'Available') {
-    Write-Log -Err "SessionHost '$SessionHostName' is not available. $($SessionHost | Out-String)"
+$DesiredStates = ('Available', 'NeedsAssistance')
+if ($SessionHost.Status -notin $DesiredStates) {
+    Write-Log -Err "SessionHost '$SessionHostName' is not in any of the desired states: $($DesiredStates -join ', ')"

```

Comment:
It would be helpful for debugging to also print the $SessionHost.Status here

Reply:
The status will printed as part of the output from this line: https://github.com/Azure/RDS-Templates/pull/442/files/5f76999770e8792707a458664aa7799d302cb3bc#diff-a16ade6c784e7c31b847bc6e8978e266R122

Reply:
But when this file is run directly as a TestScript from the Confgiuration.ps1 it won't run line 122 from  Script-RegisterSessionHost.ps1

Reply:
Of course it will run, it uses magic :) Just kidding, sorry, I completely missed that this was in `Test` script.